<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BSplineCurveWidget</class>
 <widget class="QWidget" name="BSplineCurveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addPointButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj punkt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="slidePointButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Przesuń punkt</string>
   </property>
  </widget>
  <widget class="QLabel" name="pointSizeLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Zmień rozmiar punktów</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="pointSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>48</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyczyść</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removePointButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń punkt</string>
   </property>
  </widget>
  <widget class="BSplineCurveCanvas" name="bSplineCurveCanvas" native="true">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>561</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BSplineCurveCanvas</class>
   <extends>QWidget</extends>
   <header>bsplinecurvecanvas.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
